app/views/hosts/_templates.html.erb

```diff
@@ -12,21 +12,24 @@
         <td><%= action_buttons(display_link_if_authorized(_("Edit"),
                                                           hash_for_edit_provisioning_template_path(id: template.to_param),
                                                           rel: "external"),
-                               link_to(_("Review"),
-                                       url_for(controller: '/unattended',
-                                               action: 'host_template',
-                                               kind: template.template_kind.name,
-                                               hostname: @host.name),
-                                       rel: 'external',
-                                       "data-provisioning-template": true),
-                               !Setting[:safemode_render] && link_to(_("Safemode Review"),
-                                       url_for(controller: '/unattended',
-                                               action: 'host_template',
-                                               kind: template.template_kind.name,
-                                               hostname: @host.name,
-                                               force_safemode: true),
-                                       rel: 'external',
-                                       "data-provisioning-template": true)) %>
+                               authorized_for(:permission => :view_provisioning_templates, :auth_object => template, :authorizer => authorizer) &&
```

Comment:
<%
      can_preview = authorized_for(:permission => :view_provisioning_templates, :auth_object => template, :authorizer => authorizer)
    %>

and then `can_preview &&`